features/preselecting_refund_payment_method_based_on_order.feature

```diff
@@ -0,0 +1,21 @@
+@refunds
+Feature: Preselecting refund payment method based on order's payment
+    As an Administrator
+    In order to speed up refunding process
```

Comment:
This and `As an Administrator` should be swapped :)

Reply:
May I ask why?